### PR TITLE
chore: move CI to GHA CodeBuild Runners

### PR DESCRIPTION
**Pull request by josecorella**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          NODE_OPTIONS: "--max-old-space-size=4096"
+        run: |
+          npm ci --unsafe-perm
+          npm run build
+          npm run coverage-node
+      - name: Test compliance
+        env: 
+          NODE_OPTIONS: "--max-old-space-size=4096"
+        run: |
+          npm ci --unsafe-perm
```
Comment (texastony) [inline .github/workflows/ci.yml:45]: Suggestion: Running `npm ci` once SHOULD be sufficient.
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          npm run lint
+          npm run test_conditions
+      - name: Run Test Vectors Node ${{matrix.node}}
+        env:
+          NODE_OPTIONS: "--max-old-space-size=4096"
+          NPM_CONFIG_UNSAFE_PERM: true
+          PUBLISH_LOCAL: true
+        run: |
+          npm ci --unsafe-perm
```
Comment (texastony) [inline .github/workflows/ci.yml:54]: Suggestion: Running `npm ci` once SHOULD be sufficient.
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          npm run verdaccio-publish
+          npm run verdaccio-node-decrypt
+          npm run verdaccio-node-encrypt
+      - name: Run Test Vectors Browser node ${{matrix.node}} 
+        env:
+          NODE_OPTIONS: "--max-old-space-size=4096"
+          NPM_CONFIG_UNSAFE_PERM: true
+          PUBLISH_LOCAL: true
+        run: |
+          npm ci --unsafe-perm
```
Comment (texastony) [inline .github/workflows/ci.yml:65]: Suggestion: Running `npm ci` once SHOULD be sufficient.
```suggestion
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          npm run lint
+          npm run test_conditions
+      - name: Run Test Vectors Node ${{matrix.node}}
+        env:
+          NODE_OPTIONS: "--max-old-space-size=4096"
+          NPM_CONFIG_UNSAFE_PERM: true
+          PUBLISH_LOCAL: true
+        run: |
+          npm ci --unsafe-perm
+          npm run build
```
Comment (texastony) [inline .github/workflows/ci.yml:55]: No need for an additional `build`.
```suggestion
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+          npm run verdaccio-node-decrypt
+          npm run verdaccio-node-encrypt
+      - name: Run Test Vectors Browser node ${{matrix.node}} 
+        env:
+          NODE_OPTIONS: "--max-old-space-size=4096"
+          NPM_CONFIG_UNSAFE_PERM: true
+          PUBLISH_LOCAL: true
+        run: |
+          npm ci --unsafe-perm
+          npm run build
```
Comment (texastony) [inline .github/workflows/ci.yml:66]: No need for an additional `build`.
```suggestion
```

**Review by texastony — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (josecorella) [reply .github/workflows/ci.yml:43]:
```suggestion
          npm ci
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (josecorella) [reply .github/workflows/ci.yml:52]:
```suggestion
          npm ci
```

File: .github/workflows/ci.yml (hunk 4 above)
Comment (josecorella) [reply .github/workflows/ci.yml:53]:
```suggestion
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (josecorella) [inline .github/workflows/ci.yml]:
```suggestion
```

**Review by texastony — APPROVED** [review]

**Pull request merged**